### DISCUSSION TITLE
L10S Ultra

### DISCUSSION THREAD

**Discussion by rafinesse** [merdok/homebridge-miot#398] [General]
Hi all, will L10S Ultra be supported if L10 Pro is supported? If not, could you add support for L10S Ultra?

Thank you!

**Comment by merdok** [comment]
That should not be a problem. What is the model name of the L10s ultra?

**Reply by rafinesse** [reply]
Thanks, should be RLS6LADC.

**Reply by merdok** [reply]
I would need to have the internal device model. If you try to add the vacuum already now to the plugin and then check the homebridge debug log then it should get detected and printed there. Could you please do that?

Btw, thanks for the support👍 Appreciate that!

**Reply by rafinesse** [reply]
sure thing! model is dreame.vacuum.r2228o

**Reply by rafinesse** [reply]
just tried to add it and it worked :) just need on/off to automatically start it when I shut the door, so this is totally fine. thanks for this!

**Reply by merdok** [reply]
Good to hear that it already works out of the box :)
